components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -452,12 +452,8 @@ - (void)safeAreaInsetsDidChange {
       // status bar visibility then we ignore this safeAreaInsetsDidChange event.
       return;
     }
-    CGFloat safeAreaTop = MDCDeviceTopSafeAreaInset();
-    if (_currentSafeAreaInsetTop == safeAreaTop) {
-      return;
-    }
-    _currentSafeAreaInsetTop = safeAreaTop;
```

Comment:
Why was the previous logic causing incorrect behavior?

Reply:
When the status bar is hidden the height of the AppBar gets smaller by 20 pixels on iPhone 8 and we do not want the height to change instantaneously.

Reply:
So what is the issue and why is removing this solving it? Something else is hiding the status bar and therefore the height of the flexible header is shrinking, right? (at least that's what I can see in the video). Looks like for that particular use case you want to disable the auto-adjusting of the flexible header height? I'd expect that to be addressed by turning on or off a flag that "stops" the auto-adjusting, or something like that. Let me know if I'm missing something.